### PR TITLE
Fix: Add useful utilities

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

**Comment by vg-svitla** [comment]
CI fails because there is no changes within formats. Also I put some comments to the ticket

**Review by squioc — DISMISSED** [review]
LGTM. However, I suggest some improvements

**Comment by vg-svitla** [comment]
I also added linting to the utils

File: utils/send_events.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+@app.command()
+def from_intake_formats(intake_key: str, url: str, intake_path: Path, chunk_size: int = CHUNK_SIZE):
+    events: list = read_input_messages(intake_path)
+    send_events(intake_key=intake_key, events=events, url=url, chunk_size=chunk_size)
+
+
+@app.command()
+def from_text_file(intake_key: str, url: str, file: Path, chunk_size: int = CHUNK_SIZE):
```
Comment (squioc) [inline utils/send_events.py:92]:
```suggestion
def from_text_file(intake_key: str, file: Path, url: str = "https://intake.sekoia.io/batch", chunk_size: int = CHUNK_SIZE):
```

File: utils/README.md [hunk 2]
```diff
@@ -620,4 +622,65 @@ data_sources:
...
+```
+
+### Example
+
+### From intake-formats input files
+
+Each test file `["input"]["message"]` will be sent as an event.
+
+```console
+poetry run python send_events.py from-intake-formats "<intake-key>" "<url>" ./../module/format/
```
Comment (squioc) [inline utils/README.md:653]:
```suggestion
poetry run python send_events.py from-intake-formats "<intake-key>" ./../module/format/
```

File: utils/README.md [hunk 3]
```diff
@@ -620,4 +622,65 @@ data_sources:
...
+```
+
+### From text file
+
+Each line will be sent as an event
+
+```console
+poetry run python3 send_events.py from-text-file "<intake-key>" "<url>" ../Downloads/example.txt
```
Comment (squioc) [inline utils/README.md:661]:
```suggestion
poetry run python3 send_events.py from-text-file "<intake-key>" ../Downloads/example.txt --url https://app.eur1.sekoia.io/api/v1/intake-http/batch 
```

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**